protected/controllers/PatientController.php

```diff
@@ -1544,19 +1544,19 @@ public function actionCreate()
             // not to be sync with PAS
             $patient->is_local = 1;
             
-            list($contact, $patient, $address, $referral, $patientuserreferral) = $this->performPatientSave($contact, $patient, $address, $referral, $patientuserreferral);
+            list($contact, $patient, $address, $referral, $patientuserreferral) = $this->performPatientSave($contact, $patient, $address, $referral, $patient_user_referral);
```

Comment:
You might want to change $patientuserreferral to $patient_user_referral in the list() as well